README.md

```diff
@@ -16,20 +16,31 @@ Ami app gives you flexibility to test multiple features such as :
 
 
 # Getting Started
+
 Ami app already includes all the dependencies required. To install the libraries using :
 ```
-npm install
+yarn
 ```
-**iOS only**
+
+## iOS setup
 
 iOS needs an additional step to install libraries using CocoaPods
 ```
 cd ios
 pod install
 cd ..
 ```
+
+Next, [follow these steps](https://github.com/customerio/RemoteHabits-iOS/blob/main/docs/dev-notes/DEVELOPMENT.md#setup-code-signing-internal-team-member-only) to download the provisioning profiles and certificates for development to your machine. 
+
 **Note** : Make sure that the deployment target is set to minimum 10.0 in Xcode.
```

Comment:
I think this is a mistake from my end. It should be `...minimum 13.0 in Xcode` since our SDK depends on iOS 13.0 min,